components/authz-service/server/v2/policy_test.go

```diff
@@ -2870,6 +3027,13 @@ func getPolicyFromStore(t *testing.T, store *cache.Cache, id string) storage.Pol
 	return *pol
 }
 
+func assertProjectsMatch(t *testing.T, storageProject string, apiProject string) {
+	if storageProject == constants_v2.AllProjectsID {
+		storageProject = constants_v2.AllProjectsExternalID
+	}
+	assert.Equal(t, storageProject, apiProject, "statement projects differ")
+}
```

Comment:
Needed just this one little tweak to handle the internal/external representation differences.